record/record.app.js

```diff
@@ -57,7 +58,59 @@
         context.pageId = MathUtils.uuid();
 
         ERMrest.appLinkFn(UriUtils.appTagToURL);
-
+        /* @desc getPageSize(obj) returms page size of the obj.           */
+        function getPageSize(obj){
+            return ((!angular.isUndefined(obj) && obj.display.defaultPageSize)?obj.display.defaultPageSize:constants.defaultPageSize);
+        }
+        /*@desc  getRelatedTableData(refObj, accordionOpen, callback) returns model object with all required component values
```

Comment:
This isn't being formatted like a jdoc because there isn't an extra `*`. i.e. `/** @desc ...`.